omniauth-shopify-oauth2.gemspec

```diff
@@ -16,8 +16,8 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
   s.require_paths = ['lib']
 
-  s.add_runtime_dependency 'omniauth-oauth2', '~> 1.2'
-  s.add_runtime_dependency 'oauth2', '~> 1.1.0'
+  s.add_runtime_dependency 'omniauth-oauth2', '~> 1.4.0'
```

Comment:
https://github.com/intridea/omniauth-oauth2/compare/v1.2.0...v1.4.0

The change from 1.2.0 to 1.4.0 looks safe to me but we might need to loose ruby 2.1 support. Not sure how many app which are on 2.1 still uses that gem